pom.xml

```diff
@@ -187,5 +187,20 @@
             <version>2.28.2</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
We are just changing jdk version. Do we need additional dependencies in java11 ?